public/content/lessons/2018/derivatives-and-transforms/index.mdx

```diff
@@ -1,10 +1,267 @@
+description: A visual for derivatives which generalizes more nicely to topics beyond calculus. Thinking of a function as a transformation, the derivative measure how much that function locally stretches or squishes a given region.
 date: 2018-05-19
 chapter: 15
 video: CfW845LNObM
 source: _2018/alt_calc.py
 credits:
 - Lesson by Grant Sanderson
+- Blog Adaptation by Vivek Verma
 ---
+
+## Introduction
+
+<PiCreature
+ emotion="happy"
+ text="$\frac{d(\pi r^2)}{dr} = 2 \pi r$"
+ flip={true}
+/>
+
+Picture yourself as a calculus student, about to begin your first course. The months ahead of you hold within them a lot of hard work, neat examples, not-so-neat examples, beautiful connections to physics, not-so-beautiful piles of formulas to memorize, plenty of moments getting stuck and banging your head into a wall, a few “aha” moments sprinkled in as well, and some truly lovely graphical intuitions to help guide you throughout it all. 
+
+<PiCreature
+ emotion="angry"
+ text="$\frac{d}{dx} (x^x)$"
+/>
+
+<Figure image="partial_2.png" caption="A graphical depiction of a partial differential equation."/>
+
+But if the course ahead of you is anything like my first introduction to calculus, or any of the first courses I've seen in the years since, there's one topic you will not see, but which I believe stands to greatly accelerate your learning.
+
+You see, almost of all the visual intuitions for a first year in calculus are based on graphs. The derivative is the slope of a graph, the integral is an area under that graph, etc. 
+
+<Figure image="intro_1.png" />
+
+But as you generalize calculus beyond functions whose inputs and output are simply numbers, it's not always possible to graph the function your analyzing. 
+
+<Figure image="intro_2.png" />
+
+If your intuitions for fundamental ideas like the derivative are rooted too rigidly in graphs, this can make for a very tall and largely unnecessary conceptual hurdle between you and the more “advanced” topics like multivariable calculus, complex analysis, differential geometry, etc.
+
+What I want to share with you is a way of thinking about derivatives, which I'll refer to as the *transformational view*, which generalizes more seamlessly into some of the more general contexts where calculus shows up. And then we'll use it to analyze a certain fun puzzle about repeated fractions.
+
+## The transformation view
+
+First off, let's just make sure we're all on the same page about what the standard visual is. If you graph a function which simply takes real numbers as inputs and outputs, one of the first things you learn in a calculus course is that the derivative gives you the slope of this graph. 
+
+<Figure image="transform_1.png" video="transform_1.mp4" />
+
+What we mean by that is that the derivative of the function is a new function which for every input x, returns this slope.
+
+<Question 
+  question="Quick sanity check! What's the derivative of $\sin (x) x^2$?"
```

Comment:
Given that the discussion above this was about the visual meaning of the derivative, this struck me as a somewhat out-of-place exercise. I would say either:

1) Replace it with something that shows the graph of sin(x)*x^2, and has someone estimate the derivative at a given value based looking at the slope.

2) Keep it as is, but don't frame it as a "sanity check", and instead somehow call out the fact that computing derivatives using the various derivative rules is a separate skill from the visual intuition.